app/views/shared/_accordion.html.slim

```diff
@@ -0,0 +1,16 @@
+/ [locals] ( * denotes required)
```

Comment:
I'd probably be fine without these comments. 

If we have a well-tested app, we will know that a required local var is not present. And then we won't have comments to maintain! https://robots.thoughtbot.com/letting-your-code-speak-for-itself :)